### DISCUSSION TITLE
Please help me!

### DISCUSSION THREAD

**Discussion by nxcifer** [community/community#129219] [New to GitHub]
### Select Topic Area

Question

### Body

Hello i cant open GitHub on my pc,
It allways says: "privacy 
![image](https://github.com/community/community/assets/113261990/1a6ad716-4aee-4a26-8ac5-4cfffcf6e2ea)
error" on the tab.
I tried different browsers like: opera, operagx, edge and brave. I deleted my cookiea but nothing worked.
Can someone help me?

**Comment by Carlos-Levi** [comment]
Here are some additional things to keep in mind:

The warning message in the image you sent appears to be phishing attempt. The message claims there is a problem with your connection in order to trick you into clicking on a malicious link. Be cautious about clicking on any links in the warning message.
The correct URL for GitHub is "https://github.com/".

BUT !!

The error message “Your connection is not private”  indicates that your browser is unable to verify the security certificate for GitHub. This could be due to a number of reasons, but here are a couple of things you can try:

Check your date and time settings: Make sure your computer's date and time are set correctly. A mismatch between your computer's clock and the website's security certificate can cause this error.
Clear your browsing data: Try clearing your browsing data, including cookies and cache. Corrupted data can sometimes interfere with secure connections.
Disable browser extensions: If you have any browser extensions installed, try disabling them one by one to see if one of them is causing the problem.
Check your antivirus software: Some antivirus software can interfere with secure connections. Try temporarily disabling your antivirus software to see if that resolves the issue.
If you’re still seeing the error message after trying these steps, you can try contacting GitHub support for further assistance.

**Reply by nxcifer** [reply]
I tried this with the date and time and the data but it didn't work ill try that with the antivirus

**Reply by Carlos-Levi** [reply]
Damn, I hope you can solve it

**Reply by nxcifer** [reply]
I deactivated my antivirus and even deinstalled it but it didnt worked

**Reply by nxcifer** [reply]
And its the real website it isnt a pishing website

**Reply by Carlos-Levi** [reply]
now that I realize it's the real site. Man, what's happening is so curious, I really don't know how to solve it, I hope you can

**Reply by nxcifer** [reply]
Nope i cant
